docs/articles/nunit/toc.yml

```
@@ -2,6 +2,8 @@
   href: intro.md
 - name: NUnit 4 plans
   href: Towards-NUnit4.md
+- name: NUnit 3 to 4 migration guide
```

```suggestion
- name: Migration Guides
```

This should probably be 1 page for all migration guides. We may want to backport some eventually or build upon them in 4.x -> 5.x.